packages/mui-system/src/cssVars/createCssVarsProvider.d.ts

```diff
@@ -82,7 +82,7 @@ export interface CreateCssVarsProviderResult<
         disableStyleSheetGeneration?: boolean;
       }
     >,
-  ) => React.ReactElement<any>;
+  ) => React.ReactElement<unknown>;
```

Comment:
What do you think should this type be @siriwatknp? It's the return value of `CssVarsProvider`, we should have it's props somewhere, right?

Reply:
I think it's better to change to `React.JSX.Element` based on 

<img width="603" alt="image" src="https://github.com/user-attachments/assets/e424f541-adf8-499b-8f97-545b762609f2">


https://www.typescriptlang.org/play/?ts=4.1.5#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4BYAKCrQgDsBneAFSSbgF44AKMHMBgC44Abzgw2MYUyjA6AczgBfAJScAfKKpwdcIjACuUOnAA8AE2AA3OAHp1VJVWfVXVU63YSmHAES+7dVIgA